plugins/pencilblue/templates/angular/admin/content/objects/object_form.html

```diff
@@ -1,251 +1,210 @@
-        ^angular_objects^
-        ^tmp_angular=admin=elements=is_field_valid^
-
-        $scope.hideMediaButton = true;
-
-        $scope.search = function(field) {
-            searchService.search(field.searchText, field.available_objects, ['name'], function(availableObjects) {
-                field.available_objects = availableObjects;
-                $scope.paginate(field, 0);
-            });
-        }
+angular.module('pencilblueApp', ['wysiwygElement', 'ui.sortable', 'search', 'paginate', 'parseDate', 'validation'])
+.controller('PencilBlueController', function($scope, $sce, $http, $window, $timeout, $filter, searchService, paginationService, validationService) {
+  ^angular_objects^
+  ^tmp_angular=admin=elements=is_field_valid^
+
+  $scope.hideMediaButton = true;
+
+  $scope.search = function(field) {
+    searchService.search(field.searchText, field.available_objects, ['name'], function(availableObjects) {
+      field.available_objects = availableObjects;
+      $scope.paginate(field, 0);
+    });
+  }
+
+  $scope.clearSearch = function(field) {
+    field.searchText = '';
+    $scope.search(field);
+  }
+
+  $scope.paginate = function(field, paginationIndex) {
+    field.paginationIndex = paginationIndex;
+
+    paginationService.paginate(field.available_objects, field.paginationIndex, field.paginationLimit, function(availableObjects, paginationPages) {
+      field.available_objects = availableObjects;
+      field.paginationPages = paginationPages;
+    })
+  }
 
-        $scope.clearSearch = function(field) {
-            field.searchText = '';
-            $scope.search(field);
-        }
+  $scope.pageButtonVisible = function(field, testIndex) {
+    return paginationService.pageButtonVisible(testIndex, field.paginationIndex, field.paginationPages);
+  };
 
-        $scope.paginate = function(field, paginationIndex) {
-            field.paginationIndex = paginationIndex;
+  $scope.getSortListeners = function(field) {
+    var active = $scope.customObject[field.name];
+    var inactive = field.available_objects;
 
-            paginationService.paginate(field.available_objects, field.paginationIndex, field.paginationLimit, function(availableObjects, paginationPages) {
-                field.available_objects = availableObjects;
-                field.paginationPages = paginationPages;
-            })
+    if(!active) {
+      active = [];
+    }
+
+    return {
+      accept: function (sourceItemHandleScope, destSortableScope) {
+        var itemToTest = sourceItemHandleScope.itemScope.object;
+
+        for(var i = 0; i < active.length; i++) {
+          if(itemToTest._id.toString() === active[i]._id.toString()) {
+            return true;
+          }
+        }
+        for(var i = 0; i < inactive.length; i++) {
+          if(itemToTest._id.toString() === inactive[i]._id.toString()) {
```

Comment:
Good question.  I was thinking that coming from the server the ```_id``` property was already serialized to a string.  Did that change after the mongo driver update?